elevation_layer/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package>
+  <name>elevation_layer</name>
+
+  <version>0.0.2</version>
+
+  <description>adds elevation mapping informations to costmap_2d</description>
+
+  <!--  <author email="echisari@anybotics.com">Eugenio Chisari</author>  -->
+  <maintainer email="echisari@anybotics.com">Eugenio Chisari</maintainer>
+
+  <license>BSD</license>
```

Comment:
Are we open sourcing this plugin?

Reply:
I dont know

Reply:
It's already open source, if we keep it in this repo :wink: